source/x11-helper.c

```diff
@@ -213,8 +215,66 @@ static workarea * x11_get_monitor_from_output ( xcb_randr_output_t out )
     return retv;
 }
 
+uint8_t x11_is_randr_present () {
```

Comment:
Please use `int` as return value, this is normally used in rofi for returning TRUE/FALSE;